pint/models/timing_model.py

```diff
@@ -564,7 +565,35 @@ def noise_model_basis_weight(self, toas):
             result.append(nf(toas)[1])
         return np.hstack((r for r in result))
 
-
+    def jump_flags_to_params(self, toas):
+        '''convert jump flags in toas.table["flags"] to jump parameters in the model'''
+        from . import jump
+        for dict in toas.table['flags']:
+            if 'jump' in dict.keys():
+                break
+        else:
+            print('no jump flags to process')
```

Comment:
Fixed, although there are still a few places with print statements because astropy.log.info wasn't working, even when I set the level to INFO, and I needed some function to print something to see what they were doing.